fl4health/clients/fenda_client.py

```diff
@@ -207,7 +205,7 @@ def compute_loss(
         """
 
         loss = self.criterion(preds["prediction"], target)
-        total_loss = loss
+        total_loss = loss.clone()
```

Comment:
Without clone, `total_loss` and `loss` share memory. This means that anything that is added to `total_loss` below is also added to loss. This. means that checkpoint and backward in the loss object end up being identical, which we don't want. I added a unit test to make sure the clone here fixes the issue.